packages/discord.js/typings/index.d.ts

```diff
@@ -596,6 +596,9 @@ export class ClientApplication extends Application {
   public commands: ApplicationCommandManager;
   public cover: string | null;
   public flags: Readonly<ApplicationFlags>;
+  public tags?: [string, string?, string?, string?, string?];
```

Comment:
I don't believe this is correct. Firstly, `ClientApplication#tags` is always present.

Secondly, because we default to an empty array if `tags` is not sent, `<ClientApplication>.tags[0]` would incorrectly be a string.